public/index.html

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
If you like to remove the favicon, don't forget to remove the `public/favicon.ico` file too.
But I would rather replace it.